lighthouse-cli/bin.ts

```diff
@@ -330,8 +330,14 @@ export async function runLighthouse(url: string,
     const chromeLauncher = await getDebuggableChrome(flags)
     const results = await lighthouse(url, flags, config);
 
+    // Filter out artifacts except for screenshots in traces to minimize report size.
+    var traces = results.artifacts.traces;
+    for (var pass in traces)
+      traces[pass]['traceEvents'] = traces[pass]['traceEvents'].filter(e => e['cat'] === 'disabled-by-default-devtools.screenshot');
```

Comment:
we need screenshots, and right now they're only in the trace.
we could:

1. keep this monstrosity
2. bring back the screenshots audit
3. throw screenshots into extInfo,etc  of the psi audit